lib/rules/disallow-space-after-keywords.js

```diff
@@ -51,7 +51,9 @@ module.exports.prototype = {
         );
 
         if (keywords === true) {
-            keywords = defaultKeywords;
+            keywords = defaultKeywords.filter(function(keyword) {
+                return ['function', 'return'].indexOf(keyword) === -1;
```

Comment:
Yeah even in the example it only has a few keywords (the ones that initially make sense) - not every single keyword. `if, else, for, while, do, switch, try, catch, finally` sound good